### PR TITLE
Comment out issue with linting for Nip-05

### PR DESCRIPTION
**Pull request by erskingardner**

### CODE REVIEW

**Review by pablof7z — CHANGES_REQUESTED** [review]
I think if the previous if statement has a !userProfile || !userProfile.nip05 it would remove this lint warning without non-nulling the nip05!

**Comment by erskingardner** [comment]
in the end, I needed to be explicit with an `else` but we got there.

File: ndk-svelte-components/src/lib/user/Nip05.svelte [hunk 1]
```diff
@@ -53,9 +53,12 @@
             user.fetchProfile()
                 .then(async () => {
                     userProfile = user!.profile;
-                    if (!userProfile?.nip05) reject;
-                    nip05Valid = await user!.validateNip05(userProfile?.nip05!);
-                    resolve(userProfile!);
+                    if (!userProfile || !userProfile?.nip05) {
+                        reject;
```
Comment (verbiricha) [inline ndk-svelte-components/src/lib/user/Nip05.svelte:57]: shouldn't this be a function call? `reject()` or `reject(new Error("nip-05 not found"))`. The `catch` callback function also has the same issue.

Comment (erskingardner) [reply ndk-svelte-components/src/lib/user/Nip05.svelte:57]: This code has always made me feel a little ill.... I'm going to figure out a nice/simpler way to achieve this.

**Comment by erskingardner** [comment]
Updated the code here: It's more verbose but (I think) it's much easier to follow.

**Comment by erskingardner** [comment]
Bump! @pablof7z

**Comment by erskingardner** [comment]
@pablof7z merged master – this is ready to go.

**Review by pablof7z — APPROVED** [review]

**Pull request merged**